apps/ledger-live-mobile/src/newArch/features/Web3Hub/__integrations__/web3hub.integration.test.tsx

```diff
@@ -113,4 +113,60 @@ describe("Web3Hub integration test", () => {
     await user.press(screen.getByTestId("navigation-header-back-button"));
     expect(await screen.findByText("Explore web3")).toBeOnTheScreen();
   });
+
+  it("Should list manifests, go to search and search an app and go it app page", async () => {
+    const { user } = render(<Web3HubTest />);
+
+    expect(await screen.findByText("Explore web3")).toBeOnTheScreen();
+
+    expect(await screen.findByTestId("web3hub-loading-indicator")).toBeOnTheScreen();
+    await waitForElementToBeRemoved(() => screen.getByTestId("web3hub-loading-indicator"), {
+      timeout: 1500, // timeout because we mock the request and fake 1s delay
+    });
+
+    expect(await screen.findByTestId("web3hub-main-header-search")).toBeOnTheScreen();
+    await user.press(screen.getByTestId("web3hub-main-header-search"));
+    expect(await screen.findByTestId("web3hub-search-header-search")).toBeOnTheScreen();
```

Comment:
Don't we have other alternatives than *byTestId queries?
byTestId should be used as the last resort, is it the case here? https://callstack.github.io/react-native-testing-library/docs/guides/how-to-query#idiomatic-query-predicates

Reply:
I might try the placeholder but it was not working well

Reply:
I understand your point, but maybe we should assert that the navigation did well with an assertion of something that could be seen by the end user, wdyt?

Reply:
The search input is not the same and that's what I wanted to test, also it should focus the input but I have no way of testing that I think with RTL

And I already assert other changes on navigation to make sure I'm on the correct screen as a user would